lib/command.js

```diff
@@ -42,10 +42,7 @@ module.exports = function command(opts, cb) {
   });
 
   childProcess.stdout.setEncoding('utf8');
-
-  childProcess.stdout.on('data', function(data) {
-    logger.info(data);
```

Comment:
This was being used to control log level. With this updated, `stdout` will be outputted even if logLevel is silent. It would be great to keep this functionality.